doc/source/ray-core/compiled-graph/quickstart.rst

```diff
@@ -27,10 +27,6 @@ Next instantiate the actor and use the classic Ray Core APIs ``remote`` and ``ra
     :start-after: __ray_core_usage_start__
     :end-before: __ray_core_usage_end__
 
-.. testoutput::
-
-    Execution takes 969.0364822745323 us
-
```

Comment:
FYI @ruisearch42 not sure if you were the one that wrote this, but:

1. `testoutput` needs to be associated with a `testcode` block (can't use it for `literalinclude`)
2. `testcode` needs to be python code, not bash (there were some others in these files)